src/GongSolutions.WPF.DragDrop/DefaultDropHandler.cs

```diff
@@ -181,23 +183,27 @@ public virtual void Drop(IDropInfo dropInfo)
 
             var destinationList = dropInfo.TargetCollection.TryGetList();
             var data = ExtractData(dropInfo.Data).OfType<object>().ToList();
-
+            bool forceMoveBehavior = false;
             var copyData = ShouldCopyData(dropInfo);
             if (!copyData)
             {
-                var sourceList = dropInfo.DragInfo.SourceCollection.TryGetList();
+                var sourceList = dropInfo.DragInfo.SourceCollection.TryGetList();                  
```

Comment:
Should avoid adding whitespace to the end of lines.
```suggestion
                var sourceList = dropInfo.DragInfo.SourceCollection.TryGetList();
```